CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ This file is used to list changes made in each version of the splunk cookbook.
 
 ## Unreleased
 
+- Fixes Issue [#204](https://github.com/sous-chefs/chef-splunk/issues/204)# Please enter the commit message for your changes. Lines starting
+- Addresses linting issues
+- Fixes an issue with the `user-seed.conf` file
+- Ensures that splunk is installed prior to anything in the `chef-splunk::service` recipe executes
+- Fixes Chef Delivery job in Github Actions: checks out `main` branch instead of `master` branch
```

Comment:
This should be using `main` if the upstream branch exists, otherwise they should remain using `master.

Reply:
As previously commented, IMHO, attaching to master/main leaves the outcome of the tests in a potentially unpredictable state, as the HEAD of master/main could be potentially unstable. I'd rather leave it at a specific version that has been fully tested and released knowing that the code can be stable.